src/sagemaker/workflow/steps.py

```diff
@@ -817,6 +834,13 @@ def __init__(
                 error_message="The step_args of ProcessingStep must be obtained from processor.run().",
             )
 
+            if job_arguments or inputs or outputs or code or kms_key:
```

Comment:
should we check for each individually? Otherwise the error message might get confusing